src/data_sets.h

```diff
@@ -4056,6 +4067,30 @@ typedef struct
 	/** Status (eGpxStatus) */
 	uint32_t                status;
 
+    /** GRMC BITS **/
+    uint64_t                grmcBitsSer0;
+    uint64_t                grmcBitsSer1;
+    uint64_t                grmcBitsSer2;
+    uint64_t                grmcBitsUSB;
+
+    /** Hardware status flags (eHdwStatusFlags) */
+    uint32_t                hdwStatus;
+
+    /** MCU temperature (not available yet) */
+    float					mcuTemp;
+
+	/** Preintegrated IMU (PIMU) integration period and navigation/AHRS filter output period (ms). */
```

Comment:
If navOutputPeriodMs is valid for the GPX, then its not the Preintegrated IMU period.  Let's make sure the comments on the GPX reflect the GPX functionality.